src/actions/ui.js

```diff
@@ -7,14 +7,22 @@ import {
     SET_UI_OPTIONS,
     SET_UI_FROM_VISUALIZATION,
     CLEAR_UI,
-    TOGGLE_UI_RIGHT_SIDEBAR,
+    SET_UI_DETAILS_PANEL_OPEN,
+    SET_UI_ACCESSORY_PANEL_OPEN,
     SET_UI_ACTIVE_MODAL_DIALOG,
     SET_UI_ITEMS,
     ADD_UI_PARENT_GRAPH_MAP,
     SET_UI_CONDITIONS,
     sGetUiConditions,
+    SET_UI_INPUT,
 } from '../reducers/ui.js'
 
+export const acSetUiInput = (value) => ({
+    type: SET_UI_INPUT,
+    // Store input as an object but allow passing a string for simple input types
+    value: typeof value === 'string' ? { type: value } : value,
```

Comment:
Unsure about this pattern. Its a clever solution and I get the idea, but to me its a bit over-engineered for this rather simple use-case. Would prefer if the consumer always has to provide the same params regardless of input type (i.e. `{ type: value }`), just for clarity and reduced amount of head scratching down the line 😄

Reply:
Yeah, I just figured we only need to store this as an object for the tracked entity only, but on balance, just sending the object when dispatching the action is more explicit and future proof.

Reply:
Addressed in d3aba95